client/src/components/tests/About.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Enzyme, { shallow, mount } from 'enzyme';
+
+import About from '../About';
+
+
+describe('About', () => {
+  const props = {
+    stockInfo: [
+      {
+        CEO: '',
```

Comment:
can do additional testing with some filled out stockInfo and test if it matches in the shallow render.